src/inmanta/agent/agent_new.py

```diff
@@ -28,17 +28,17 @@
 from inmanta.agent import executor, forking_executor
 from inmanta.agent.reporting import collect_report
 from inmanta.const import AGENT_SCHEDULER_ID
-from inmanta.data.model import DataBaseReport
+from inmanta.data.model import DataBaseReport, SchedulerStatusReport
 from inmanta.deploy import scheduler
-from inmanta.protocol import SessionEndpoint, methods, methods_v2
+from inmanta.protocol import SessionEndpoint, TypedClient, methods, methods_v2
 from inmanta.server.services.databaseservice import DatabaseMonitor
 from inmanta.types import Apireturn
 from inmanta.util import ensure_directory_exist, join_threadpools
 
 LOGGER = logging.getLogger("inmanta.scheduler")
 
 
-class Agent(SessionEndpoint):
+class Agent(SessionEndpoint, TypedClient):
```

Comment:
1. can you do this? 
2. What does this mean?

Reply:
Removed, it was an artifact from experimentation with `TypedClient`